server/index.js

```diff
@@ -58,9 +63,9 @@ app.set("views", "./views");
 app.use(cookieParser());
 app.use(bodyParser.json());
 app.use(
-  express.urlencoded({
-    extended: true
-  })
+    express.urlencoded({
+        extended: true
+    })
 ); //key=value&key=value
```

Comment:
wat is deze

Reply:
Heb ik niet gemaakt. Format is alleen aangepast